spring-cloud-contract-stub-runner/pom.xml

```diff
@@ -172,6 +179,60 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
```

Comment:
This should shade the dependencies but for some reason it seems that on the consumer side still original classes are called